HOW_TO.md

```diff
@@ -0,0 +1,53 @@
+## How to join the network?
+
+RPC address would be provided here shortly.
+
+We recommend you to use [near-cli documentation](https://docs.near.org/tools/near-cli), e.g. how to [create an account](https://docs.near.org/tools/near-cli#near-create-account).
+
+StatelessNet is a sandbox created for load testing purposes.
+The traffic from mainnet may be forwarded to StatelessNet.
```

Comment:
nit: The StatelessNet will be initiated with a copy of mainnet state adjustments to allow the dev full control of the newtork. In the future the protocol team may enable mirroring mainnet traffic in StatelessNet.